server/src/test/java/com/cloud/vm/UserVmManagerImplTest.java

```diff
@@ -561,12 +561,13 @@ private void prepareAndRunConfigureCustomRootDiskSizeTest(Map<String, String> cu
     public void verifyIfHypervisorSupportRootdiskSizeOverrideTest() {
         Hypervisor.HypervisorType[] hypervisorTypeArray = Hypervisor.HypervisorType.values();
         int exceptionCounter = 0;
-        int expectedExceptionCounter = hypervisorTypeArray.length - 4;
+        int expectedExceptionCounter = hypervisorTypeArray.length - 5;
 
         for(int i = 0; i < hypervisorTypeArray.length; i++) {
             if (Hypervisor.HypervisorType.KVM == hypervisorTypeArray[i]
                     || Hypervisor.HypervisorType.XenServer == hypervisorTypeArray[i]
                     || Hypervisor.HypervisorType.VMware == hypervisorTypeArray[i]
+                    || Hypervisor.HypervisorType.Custom == hypervisorTypeArray[i]
                     || Hypervisor.HypervisorType.Simulator == hypervisorTypeArray[i]) {
```

Comment:
```suggestion
            if (Hypervisor.HypervisorType.isSupported(hypervisorTypeArray[i]) {
```
or somthing like that.

Reply:
Fixed